### ISSUE TITLE
Google Chrome cannot read and write to its data directory : profile

### ISSUE THREAD

**Issue by Elanuroquesalta** [G33kDude/Chrome.ahk#26]
When i run any of the examples scripts or an external script using the library a message pops out saying:

"Google Chrome cannot read and write to its data directory : profile"

![image](https://user-images.githubusercontent.com/92059738/136282514-0b6f13da-7db1-49ef-94cb-b1b7985d9b0c.png)

Any ideas how to fix it?

**Comment by SaifAqqad** [comment]
Passing the profile folder as a full path fixed it for me
```ahk
ChromeInst := new chrome(A_ScriptDir "\profile", "https://autohotkey.com")
```

Also you shouldn't name the variable `chrome` because that's the class's name and classes are actually just objects that are stored in a super-global variable, and in this case, that variable is named `chrome`. [AHK docs](https://www.autohotkey.com/docs/Objects.htm#Custom_Classes)

**Comment by Elanuroquesalta** [comment]
> Passing the profile folder as a full path fixed it for me
> 
> ```ahk
> ChromeInst := new chrome(A_ScriptDir "\profile", "https://autohotkey.com")
> ```
> 
> Also you shouldn't name the variable `chrome` because that's the class's name and classes are actually just objects that are stored in a super-global variable, and in this case, that variable is named `chrome`. [AHK docs](https://www.autohotkey.com/docs/Objects.htm#Custom_Classes)


MUCH THANKS! 

copying as a full path fixed for me aswell, also thank you for pointing out the class name thing.

I will continuing working on, cheers :)

**Comment by G33kDude** [comment]
This should be fixed in 1.3.0
